docs/stylesheets/ft.extra.css

```
@@ -15,3 +15,7 @@
 .md-version__list {
     font-weight: 500 !important;
 }
+
+#available-endpoints + p + .md-typeset__scrollwrap .md-typeset__table th:first-of-type {
```

This looks VERY fragile

simply adding another paragraph will break this completely ... is it possible to have a wildcard between the `+` characters instead?

---

```suggestion
#available-endpoints ~ .md-typeset__scrollwrap .md-typeset__table th:first-of-type {
```

this seems to work - but can you try this on your side, too?

---

Oh nice alternative - yeah I felt that was flaky but didn't have a nicer suggestion. Thanks!

---

msdn docs for the rescue :rofl: